db/migrate/20230117153936_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.string :posts_counter
```

Comment:
- [ ] Counter should be an integer, not a string. An integer is a numeric value, while String is a character value represented in quotes. Let's kindly rectify the issue.